components/admin/lessons/AdminLessonCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Link from 'next/link'
+import styles from '../../../scss/adminLessonCard.module.scss'
+
+type Props = {
+  heading: string
+  description: string
+  pendingFlaggedQuestions: number
+  editLessonUrl: string
+  width?: string
```

Comment:
why are width and height strings?

Reply:
I thought in React when you set it to a `px` or `%` it needed to be in a string format